layers/bexley/text-sld.inc

```diff
@@ -0,0 +1,55 @@
+CLASS
+    EXPRESSION "1"
+    LABEL
+        FONT [font]
+        TYPE TRUETYPE
+        COLOR 0 0 0
+        SIZE [height]
+        POSITION [position]
+        ANGLE [orientdeg]
+    END
+END
+CLASS
+    EXPRESSION "2"
+    LABEL
+        FONT [font]
+        TYPE TRUETYPE
+        COLOR "#0099ff"
+        SIZE [height]
+        POSITION [position]
+        ANGLE [orientdeg]
+    END
+END
+#CLASS
```

Comment:
Has this been left out deliberately?

Reply:
Yeah; 3 is only trig point heights, looks like, and it's not plotting the trig points. And 5 was boundary text, when it's also not plotting the boundaries.